TimelineSection.js

```diff
@@ -0,0 +1,273 @@
+              Research and program development
+            </span>
+            {/* <LeftBlob style = {StylesProvider.overlay/> */}
+            <p className={classes.eventContent}>
+              Toronto Tech Mentoring was born out of the Civic Tech Toronto
+              space. Civic Tech is a movement that focuses on developing and
+              harnessing technology for the betterment of civic life.
+            </p>
+            <div>
+              <Pic1
+                className={`${classes.firstPic} ${classes.pic}`}
+                // src={pic1}
+              />
+              <Pic1 className={classes.pic}/>
+              {/* <img className={classes.bg} alt="purple bubble" src={bg3} /> */}
+            </div>
+          </Grid>
+        </Grid>
+        <Grid className={classes.gridContainer} container xs={12}>
+          <Grid item xs={2} md={3} lg={3} className={classes.center}>
+            <span className={classes.yearTag}>2018</span>
+            <Pin className={classes.pinStyle} />
+
+            {/* <img className={classes.pinStyle} alt="pin" src={pin} /> */}
+          </Grid>
+          <Grid container item xs={10} md={9} lg={9}>
+            <span className={classes.yearTagSm}>2018</span>
+            <span className={classes.eventTitle}>
+              Pilot testing and refinement
+            </span>
+            <p className={classes.eventContent}>
+              Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean ac
+              mi porta, finibus nisl a, gravida massa. Etiam non quam sit amet
+              nisi tristique viverra at ac elit.
+            </p>
+            <div>
+              <Pic3
+                className={`${classes.firstPic} ${classes.pic}`}
+                // src={pic3}
+              />
+              <Pic3 className={classes.pic}/>
+              {/* <img className={classes.bg} alt="blue bubble" src={bg2} /> */}
+            </div>
+          </Grid>
+        </Grid>
+        <Grid className={classes.gridContainer} container xs={12}>
+          <Grid item xs={2} md={3} lg={3} className={classes.center}>
+            <span className={classes.yearTag}>2019</span>
+            <Pin className={classes.pinStyle} />
+
+            {/* <img className={classes.pinStyle} alt="pin" src={pin} /> */}
+          </Grid>
+          <Grid item xs={10} md={9} lg={9}>
+            <span className={classes.yearTagSm}>2019</span>
+            <span className={classes.eventTitle}>Full program launch!</span>
+            <p className={classes.eventContent}>
+              Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean ac
+              mi porta, finibus nisl a, gravida massa. Etiam non quam sit amet
+              nisi tristique viverra at ac elit.
+            </p>
+            <Grid item xs={12}>
+              {/* <img className={classes.bg} alt="orange bubble" src={bg3} /> */}
+            </Grid>
+          </Grid>
+        </Grid>
+      </Grid>
+    )
```

Comment:
is this file part of this sidebar issue? if not, better to merge it separately, right?